.elasticbeanstalk/config.yml

```diff
@@ -1,11 +1,11 @@
 branch-defaults:
   master:
-    environment: Server-env-1
+    environment: Server-env
 deploy:
   artifact: build/libs/server-1.0-SNAPSHOT-all.jar
 global:
   application_name: server
-  default_ec2_keyname: e-fts2bpcxsi
+  default_ec2_keyname: e-qaxcb3z4ga
```

Comment:
Can you elaborate on why you need to change the existing EC2 instance? I have a gap in my knowledge around Elasticbeanstalk and how it works.

Reply:
I approached as separate components of one application.  I have two EC2 instances under one server application.  The first instance contains the echoserver portion, while this new EC2 instance contains the http portion.  Does that make sense?